internal/services/instances.go

```diff
@@ -374,7 +374,13 @@ func (s *ServiceClient) DeleteForHelmController(ctx context.Context, namespace,
 		metav1.DeleteOptions{},
 	)
 
-	return errors.Wrap(err, "error deleting helm charts")
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			return nil
+		}
+	}
```

Comment:
From the `IsNotFound` doc

```
// It supports wrapped errors and returns false when the error is nil.
```

so I guess that we can drop the `err != nil` check.

```suggestion
	if apierrors.IsNotFound(err) {
		return nil
	}
```